nixos/modules/tasks/filesystems/overlayfs.nix

```diff
@@ -73,10 +76,22 @@ let
         description = ''
           The path to the workdir.
 
+          If the filesystem is `neededForBoot`, this will be prefixed with `/sysroot`,
+          unless `prefixWithSysroot` is set to `false`.
+
           This MUST be set if you set `upperdir`.
         '';
       };
 
+      prefixWithSysroot = lib.mkOption {
+        type = lib.types.bool;
+        default = true;
+        description = ''
+          If enabled, `upperdir` and `workdir` will be prefixed with `/sysroot`.
```

Comment:
Applies to lowerdir too though, right? Sorry, should have caught that the first time around.

Reply:
Right, thank you. I should've noticed that it's prefixed too while writing the doc here. Sorry